_scripts/icon_list_refresh.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
 
+if [ -z $1 ]
+then
+  echo "Provide the path to the TinyMCE source root directory"
+  exit 1
+else
+  tinymce_root=$1
+fi
 # Output markdown file
 list_file='_includes/configuration/icon_list.md'
 # Input directory containing the icon SVGs
 icon_svg_dir='images/icons/'
+rm -rf $icon_svg_dir*
```

Comment:
There's a check just below this in the script to see whether we're running the script in the wrong directory. Should this section be moved to after that check, in case we accidentally `rm -rf` the wrong thing? (Also, would `rm -r` work? I'm not a big fan of squashing the errors in a destructive command if we can avoid it).